plugin/evm/orderbook/matching_pipeline.go

```diff
@@ -86,6 +86,9 @@ func (pipeline *MatchingPipeline) Run(blockNumber *big.Int) bool {
 	}
 
 	orderBookTxsCount := pipeline.lotp.GetOrderBookTxsCount()
+	if orderBookTxsCount > 0 {
+		pipeline.lotp.SetOrderBookTxsHeadBlockNumber(blockNumber.Uint64())
+	}
 	return orderBookTxsCount > 0
```

Comment:
to make it slightly more cleaner, consider returning `true` on line 91 and `false` on line 92.

Reply:
done now